modules/deleting-an-accelerator-profile.adoc

```diff
@@ -0,0 +1,34 @@
+:_module-type: PROCEDURE
+
+[id="deleting-an-accelerator-profile_{context}"]
+= Deleting an accelerator profile
+
+[role='_abstract']
+To discard accelerator profiles that you no longer require, you can delete them so that they do not appear on the dashboard.
+
+.Prerequisites
+* You have logged in to {openshift-platform}.
+* You have the `cluster-admin` role in {openshift-platform}.
+* The `acceleratorprofiles.dashboard.opendatahub.io` custom resource definition (CRD) exists in your deployment and contains an accelerator profile.
+
+.Procedure
+. In the {openshift-platform} web console, in the *Administrator* perspective, click *Administration* -> *CustomResourceDefinitions*.
+. Click the `acceleratorprofiles.dashboard.opendatahub.io` custom resource definition (CRD).
++
+A details page for the custom resource definition (CRD) opens.
+. Click the *Instances* tab.
+. Click the action menu (*&#8942;*) beside the accelerator profile that you want to delete and click *Delete*.
```

Comment:
The button says 'Delete AcceleratorProfile'

Reply:
Fixed.